extensions/wikibase/module/scripts/preview-renderer.js

```diff
@@ -312,6 +312,10 @@ EditRenderer._renderValue = function(json, container) {
   
   if (mode === "wikibase-item" || mode === "wikibase-property") {
     EditRenderer._renderEntity(json.datavalue, container);
+  } else if (json.snaktype === "somevalue") {
+    $('<span>somevalue</span>').appendTo(container);
+  } else if (json.snaktype === "novalue") {
+    $('<span>novalue</span>').appendTo(container);
```

Comment:
Since those strings are shown to the user, it would be good to:
- use user-friendly terms instead (Wikibase displays "unknown value" / "no value")
- make it possible to localize them with `$.i18n`

Reply:
+1 I would suggest "some value" and "no value" as Commons does it.